functions/system.bash

```diff
@@ -213,6 +213,34 @@ misc_system_settings() {
   echo "OK"
 }
 
+## change system swap size dependent on free space on /
+## swap on SD (/var/swap per default) only used after ZRAM swap full if that exists
+##
+##    change_swapsize(int size in MB)
+##
+change_swapsize() {
+  local totalMemory
+
+  totalMemory="$(grep MemTotal /proc/meminfo | awk '{print $2}')"
+  if [ -z "$totalMemory" ]; then return 1; fi
+
+  swap=$((2*totalMemory))
+  minfree=$((2*swap))
+  free=$(df -hk / | awk '/dev/ { print $3 }')
+  if [ "$free" -ge "$minfree" ]; then
+    size=$swap
+  elif [ "$free" -ge "$swap" ]; then
+    size=$totalMemory
+  else
+    return 0
+  fi
+
+  if is_raspbian; then
```

Comment:
Return at begin of function of !Raspbian. Add echo setting swap size to ... MB.

Reply:
done